### ISSUE TITLE
Acrogeric hands

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11500]
**Preferred term label:**

**Synonyms**
Acrogeria
Prematurely aged hands
Aged appearance of the hands
Thinning and wrinkling of the skin of the hands

**Definition (free text, please give PubMed ID)**

The Ehlers-danlos society (https://www.ehlers-danlos.com/cleds/)   Prematurely aged appearance of the hands: People with clEDS may notice that the skin on their hands has a prematurely aged appearance. 

PMID: 28306229 and PMID: 32572181 The major clinical criteria of clEDS are (1) skin hyper extensibility, with velvety skin texture and absence of atrophic scarring; (2) generalized joint hypermobility with or without recurrent dislocations (most commonly shoulder and ankle); and (3) easy or spontaneous bruising of skin.
Minor criteria are (1) foot deformities including broad/plump forefoot, brachydactyly with excessive skin, pes planus, hallux valgus, and piezogenic papules; (2) edema in the legs in absence of cardiac failure; (3) mild proximal and distal muscle weakness; (4) axonal polyneuropathy; (5) atrophy of muscles in hands and feet; (6) **acrogeric hands**, mallet finger(s), clinodactyly, and brachydactyly; and (7) vaginal/uterus/rectal prolapse

PMID: 36108117 Minor criteria Classical-LikeEDS(clEDS)
 Hand anomalies including:
⚬ **Acrogeric hands** (characterized by thinning and wrinkling of the skin) with excessive skin
⚬ Mallet finger(s) 
⚬ Clinodactyly
⚬ Brachydactyly

Minor criteria Periodontal EDS (pEDS)
– Easy bruising
– Joint hypermobility, mostly distal joints
– Skin hyperextensibility and fragility, abnormal scarring (wide oratrophic)
– Increased rate of infections
– Hernias
– Marfanoid facial features
– **Acrogeria**
– Prominent vasculature

**Parent term (use hpo.jax.org/app)**
Abnormal hand morphology HP:0005922

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Classical-like Ehlers-Danlos syndrome type 1 ORPHA:230839
Periodontal Ehlers-Danlos syndrome ORPHA:75392

**Your nano-attribution (ORCID)**
0009-0005-6714-5727
